src/app/@ansyn/ansyn/modules/overlays/components/overlay-timeline-status/overlay-timeline-status.component.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('OverlayTimelineStatusComponent', () => {
 		expect(component).toBeTruthy();
 	});
 
-	it('Should change "showStatus" to true only if statusMessage exists and not loading', async(async () => {
+	it('Should change "showStatus" to true only if statusMessage exists and not loading', waitForAsync(async () => {
```

Comment:
Seems like duplicate call? async(async(...))

Reply:
the first function is the `waitForAsync`, which is the replacement for the previous `async` from angular.
the second `async` is the base js async